### PR TITLE
Feature form tab styling

### LINKED ISSUE #1151: Nicer tab widget
Instead of this

image

Let's have this

image

Notes:
- we should keep in mind that people may have end up with many tabs, or long tab names - there should be a way to scroll through them
- probably it does not need to be completely different tab widget, we just need to restyle it, but keep the behavior (?)
- the same widget could be used also to switch between different tabs in projects list

### PR DESCRIPTION
**Pull request by sklencar**
* border tab has the same color as labels (color picker shows me some weird color from the design image on dropbox - #7A7368)
* from a design, the tab toolbar's height seems to be the same as the one header has. However, now is set to 90% of the rowHeight. This ratio is rather random though, couldn't find any reference about its height.

Generally, fonts have to be adjusted to Rado's design.

closes #1151 
IMG_863AAC526DB3-1

### CODE REVIEW

**Comment by wonder-sk** [comment]
> Shall we add also opacity to inactive/disable tabs as it is in the design?

Yes please :-)

Few other bits:
- font is the same for active / inactive tabs (lato bold)
- there's a darker line on top and bottom of the tab widget
- a bit smaller vertical height of the tab widget (like in the design)

**Comment by sklencar** [comment]
Other bits resolved + updated screenshot

File: qgsquick/from_qgis/plugin/qgsquickfeatureformstyling.qml [hunk 1]
```diff
@@ -74,8 +76,8 @@ QtObject {
       property int fontPixelSize: 48 * QgsQuick.Utils.dp
       property real sideMargin: 12 * QgsQuick.Utils.dp // left or right margin for a field's content
       property real outerMargin: 20 * QgsQuick.Utils.dp // left or right margin for a whole field component
-      property int fontPointSize: 16 * QgsQuick.Utils.dp
-      property int labelPointSize: 14 * QgsQuick.Utils.dp
+      property int fontPointSize: 16
+      property int labelPointSize: 14
```
Comment (tomasMizera) [inline qgsquick/from_qgis/plugin/qgsquickfeatureformstyling.qml:80]: True, this should probably not be multiplied by Dp constant, though on Android I feel like the text size is too small then. What if we would rather use pixel sizes? They can be mutliplied and should scale fine on all platforms then.

**Review by tomasMizera — COMMENTED** [review]
I really like the new form and all these upgrades! 

Is this still WIP PR?

File: qgsquick/from_qgis/plugin/qgsquickfeatureformstyling.qml (hunk 1 above)
Comment (sklencar) [reply qgsquick/from_qgis/plugin/qgsquickfeatureformstyling.qml:80]: Well, pointSize should be more reliable, but it seems to be not. I think it is not really visible from the second screenshot, but also according some forum threads, pointSize doesn't guarantee uniformity for all devices. I think PeterP was dealing with this at the very beginning and we ended up with pixel sizes and dp multiplier.

So, maybe we can ask Rado to give us also pixelSize for all text formats.

**Review by tomasMizera — APPROVED** [review]
Sorry, looking closer, font size seems the same as it was before. I am ok to merge :)

**Pull request merged**